src/components/svgicon/SVGIcon.stories.mdx

```diff
@@ -91,7 +91,10 @@ import CopyToClipboard from "react-copy-to-clipboard";
 
 <Preview>
   <Story name="ui">
-    <Flex justifyContent="space-evenly" style={{flexWrap: "wrap"}}>
+    <Flex justifyContent="space-evenly" style={{ flexWrap: "wrap" }}>
```

Comment:
`css={{ flexWrap: "wrap" }}`

Reply:
CSS not working in the current configuration